*** mariadb-server-5.5.postinst	2013-02-28 09:41:43.969994431 +0200
--- ../../mariadb-server-5.5.postinst	2013-02-28 09:41:43.969994431 +0200
*************** set_mysql_rootpw() {
*** 33,35 ****
         db_set mysql-server/root_password ""
-        db_set mysql-server/root_password_again ""
  
--- 33,34 ----
*************** EOF
*** 215,217 ****
          "USE mysql;\n" \
!         "CREATE TABLE plugin (name char(64) COLLATE utf8_bin NOT NULL DEFAULT '', " \
          "  dl char(128) COLLATE utf8_bin NOT NULL DEFAULT '', " \
--- 214,216 ----
          "USE mysql;\n" \
!         "CREATE TABLE IF NOT EXISTS plugin (name char(64) COLLATE utf8_bin NOT NULL DEFAULT '', " \
          "  dl char(128) COLLATE utf8_bin NOT NULL DEFAULT '', " \
*************** EOF
*** 231,247 ****
      set -e
- 
-     # If there is a real AppArmor profile, we reload it.
-     # If the default empty profile is installed, then we remove any old
-     # profile that may be loaded.
-     # This allows upgrade from old versions (that have an apparmor profile
-     # on by default) to work both to disable a default profile, and to keep
-     # any profile installed and maintained by users themselves.
-     profile="/etc/apparmor.d/usr.sbin.mysqld"
-     if [ -f "$profile" ] && aa-status --enabled 2>/dev/null; then
-         if grep -q /usr/sbin/mysqld "$profile" 2>/dev/null ; then
-             apparmor_parser -r "$profile" || true
-         else
-             echo "/usr/sbin/mysqld { }" | apparmor_parser --remove 2>/dev/null || true
- 	fi
-     fi
    ;;
--- 230,231 ----
